ui/pages/institutional/interactive-replacement-token-page/interactive-replacement-token-page.test.tsx

```diff
@@ -73,17 +83,13 @@ const connectRequests = [
     labels,
     origin: 'origin',
     environment: 'environment',
+    token: 'token1',
   },
 ];
 
-const props = {
-  history: {
-    push: jest.fn(),
-  },
-};
-
-const render = ({ newState } = {}) => {
+const render = ({ newState = {} } = {}) => {
   const mockSelectedInternalAccount = {
+    // @ts-expect-error: todo: fix mockState ts(2345) missing properties expected by the function
```

Comment:
Is there a ticket capturing this TODO?

Reply:
No, there isn't. Where I can create tickets in MM?

Reply:
you can directly create in https://github.com/MetaMask/metamask-extension/issues and assign/notify relative teams

Reply:
Ticket created: https://github.com/MetaMask/metamask-extension/issues/26159